mako.systemd.service.in

```diff
@@ -6,6 +6,7 @@ PartOf=graphical-session.target
 [Service]
 Type=dbus
 BusName=org.freedesktop.Notifications
+ExecCondition=/bin/sh -c '[[ "$XDG_SESSION_TYPE" == "wayland" ]]'
```

Comment:
`[[` is a bashism, we should use `[` instead.

Can we check that `WAYLAND_DISPLAY` is non-empty instead? Otherwise starting Sway from a TTY will set `XDG_SESSION_TYPE` to `tty`, see https://github.com/systemd/systemd/issues/14489.

Reply:
You sure? It seemed to work as-is, and it points to `sh`, not `bash`.

Reply:
`/bin/sh` is often times a symlink to `/bin/bash` unfortunately.

You can test with a POSIX shell if you want (e.g. https://github.com/emersion/mrsh, shameless ad). ShellCheck should also show an error.

Reply:
Let me know if this condition looks okay, there might be a clearer way in sh, but I'm not that proficient :sweat: .

Reply:
```
➜ sh --version
GNU bash, version 5.0.16(1)-release (x86_64-pc-linux-gnu)
```

I was convinced it wasn't :sweat_smile: